bitnami/discourse/values.yaml

```diff
@@ -312,34 +316,46 @@ persistence:
   ##   GKE, AWS & OpenStack)
   ##
   # storageClass: "-"
-  ##
+
   ## If you want to reuse an existing claim, you can pass the name of the PVC using
   ## the existingClaim variable
   # existingClaim: your-claim
-  accessMode: ReadWriteOnce
+  ## To use the portal and to ensure you can scale Discourse you need to provide a
+  ## ReadWriteMany PVC, if you don't have a provisioner for this type of storage
+  ## we recommend that you install the NFS provisioner and map it to a RWO volume
+  ## helm install nfs-server stable/nfs-server-provisioner --set persistence.enabled=true,persistence.size=10Gi
+  ##
+  accessMode: ReadWriteMany
```

Comment:
You are totally right here, but this Chart already offers a `values-production.yaml` file oriented to production-ready environments, which considers `ReadWriteMany` as the default value for this property in order to scale up. However, sometimes it is difficult to find a provisioner for this type and `values.yaml` takes that into account, easing the deployment when the number of replicas is 1.

Reply:
done